generators/pkg/parser/parser_test.go

```diff
@@ -0,0 +1,27 @@
+package parser
+
+import (
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestParse(t *testing.T) {
+	file, err := os.Open("../../testdata/products.yaml")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	got, err := Parse(file)
+	if err != nil {
+		t.Fatalf("Error from 'Parse': %v", err)
```

Comment:
very small nit: Error should not be capitalized. See https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Got it! Already fix it!